openadapt/record.py

```diff
@@ -167,6 +170,7 @@ def process_events(
     started = False
     while not terminate_processing.is_set() or not event_q.empty():
         event = event_q.get()
+        window_event_id = event.data["window_id"]
```

Comment:
Not all events will have a window_id

Reply:
Done in [8dabc63](https://github.com/OpenAdaptAI/OpenAdapt/pull/842/commits/8dabc634da9fe79fcee89d4bd9eed64bd81613b4)